ROADMAP.md

```diff
@@ -0,0 +1,47 @@
+- Promise Play
+  In addition to making sure that a Promise is always returned from play(), if Promises are supported or polyfilled, we should be able to shim a play() promise if a promise isn't provided to us natively.
+  https://github.com/videojs/video.js/pull/5227
+  https://github.com/videojs/video.js/issues/3927
+- vjs-fill being first class
+  vjs-fluid is a first-class option now but vjs-fill is just as useful. We should have a fill() method and option just like fluid
+- loadMedia: https://github.com/videojs/video.js/issues/4342
+- refocus docs on handwritten docs rather than jsdoc using docusaurus or docz
+- turn video.js into a monorepo and break it up into many modules
+  especially with the addition of VHS, a lot of times you want just a piece of Video.js. We should slice video.js up into pieces and publish them as separate packages. This may make the need for "custom builds" obsolete
```

Comment:
Should VHS source code be then moved into the video.js monorepo?

Reply:
can someone point me to a video.js document describing this 'VHS'? I keep seeing it but not sure what it is.

Reply:
heh, it's what we call the replacement of contrib-hls https://github.com/videojs/http-streaming